### ISSUE TITLE
still not working here - it crashes Hassio -

### ISSUE THREAD

**Issue by gnkarn** [hassio-addons/addon-vscode#13]
# Problem/Motivation
it just hangs after starting , it shows the files structure on the left , but by clicking any , it wont open and remains there " waiting" .
then , the addon simply crashes the  home assistant. 

> (Why the issue was filed)

## Expected behavior

> (What you expected to happen)

## Actual behavior
after rebooting , received this log error
```
ModuleNotFoundError: No module named 'distutils.util'
[10:19:09] FATAL: Failed executing init command: pip3 install yamllint
[cont-init.d] 81-user-scripts.sh: exited 1.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
-----------------------------------------------------------
                Oops! Something went wrong.

 We are so sorry, but something went terribly wrong when
 starting or running this add-on.
 
 Be sure to check the log above, line by line, for hints.
-----------------------------------------------------------
[cont-finish.d] 99-message.sh: exited 0.
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
```
> (What actually happened)

## Steps to reproduce

> (How can someone else make/see it happen)

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by frenck** [comment]
`FATAL: Failed executing init command: pip3 install yamllint` Why are you trying to install yamllint? it is already build-in....

**Comment by frenck** [comment]
Since you've opened #15 @gnkarn, this one can be closed?
I'm lost on what you are trying and or doing at this point.

**Comment by frenck** [comment]
Closed, see #15
